sources/ca/on/lasalle.json

```diff
@@ -11,6 +11,12 @@
                 "name": "county",
                 "protocol": "http",
                 "data": "https://data.openaddresses.io/cache/ca-on-lasalle.geojson",
+                "website": "https://data.openaddresses.io/cache/ca-on-lasalle.geojson",
```

Comment:
Is there a better website?

Reply:
> Is there a better website?

Not that I've come across... and now that I look at it, the license link takes us to the County of Essex, which maps the same polygon as Lasalle and has it's own source file...